src/app/features/genes/components/gene-evidence-proteomics/gene-evidence-proteomics.component.ts

```diff
@@ -160,31 +145,23 @@ export class GeneEvidenceProteomicsComponent {
         });
 
         if (data) {
-          this.processDifferentialExpressionData(item, data, this.LFQYAxisMin, this.LFQYAxisMax, proteomicData);
+          if (!this.LFQRange)
+            this.LFQRange = new ChartRange(data.min, data.max);
```

Comment:
Makes more sense to initialize the chart range with the data min and max values and compare against log2_fc values in the processDifferentialExpressionData function.